Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1490,24 +1500,30 @@ static HRESULT __CGContextRenderImage(CGContextRef context, ID2D1Image* image) {
 
 static HRESULT __CGContextDrawGeometry(CGContextRef context, ID2D1Geometry* geometry, CGPathDrawingMode drawMode) {
     ComPtr<ID2D1CommandList> commandList;
-    HRESULT hr = __CGContextRenderToCommandList(context, &commandList, [geometry, drawMode](CGContextRef context, ID2D1DeviceContext* deviceContext) {
-        auto& state = context->CurrentGState();
-        if (drawMode & kCGPathFill) {
-            if (drawMode & kCGPathEOFill) {
-                // TODO(DH): GH#1077 Regenerate geometry in Even/Odd fill mode.
-            }
-            deviceContext->FillGeometry(geometry, state.fillBrush.Get());
-        }
-
-        if (drawMode & kCGPathStroke && std::fpclassify(state.lineWidth) != FP_ZERO) {
-            // This only computes the stroke style if its parameters have changed since the last draw.
-            state.ComputeStrokeStyle(deviceContext);
-
-            deviceContext->DrawGeometry(geometry, state.strokeBrush.Get(), state.lineWidth, state.strokeStyle.Get());
-        }
-
-        return S_OK;
-    });
+    HRESULT hr = __CGContextRenderToCommandList(context,
```

Comment:
This is Clang :(

---
In reply to: [86919144](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r86919144) [](ancestors = 86919144)